### ISSUE TITLE
Amplitude-to-decibel conversion produces different results on different batches

### ISSUE THREAD

**Issue by jtcramer** [keunwoochoi/kapre#46]
Related to #16, I found another issue that contributes to different prediction results depending on batch size (and the batches themselves). In particular, it occurs when using converting spectrograms to decibels.

https://github.com/keunwoochoi/kapre/blob/master/kapre/backend_keras.py#L17

The maximum is taken over the entire tensor, instead of per example in the batch. This results in different normalization when the examples in a batch are different.

**Comment by keunwoochoi** [comment]
Oh, you're correct, I didn't realise it. Thanks! In the next version (which tbh I'm not sure when I'd release) I'll fix it.

**Comment by loretoparisi** [comment]
that would be awesome I have the same problem since I'm converting to decibel too!

**Comment by keunwoochoi** [comment]
Ok, what a delay for such a small change :) [this branch] has fixed, sample-wise `max()`. Will merge later when it seems fine. 

Memo for me: DO NOT FORGET TO UPDATE THE VERSION

**Comment by keunwoochoi** [comment]
..and #44.

**Comment by jtcramer** [comment]
@keunwoochoi Would it be possible to make a new release soon? We're hoping to make a new release of one of our repos that depend on this issue being fixed as soon as we can.

**Comment by justinsalamon** [comment]
+1

**Comment by keunwoochoi** [comment]
Thanks for interests! version 0.1.4 fixes the problem and is on here/pip (8090823)
